### PR TITLE
BigQuery: rewrite simple app tutorial.

### PR DESCRIPTION
**Pull request by tswast**
- Add additional region tags to support tutorial rewrite.
- Use nicer struct read for printing results.
- Use more relevant query from public datasets.

### CODE REVIEW

File: bigquery/simpleapp/simpleapp.go [hunk 1]
```diff
@@ -35,28 +37,41 @@ func main() {
...
+				'https://stackoverflow.com/questions/',
+				CAST(id as STRING)) as url,
+			view_count
+		FROM ` + "`bigquery-public-data.stackoverflow.posts_questions` " +
+		`WHERE tags like '%google-bigquery%'
+		ORDER BY view_count DESC
+		LIMIT 10;`)
 	// Use standard SQL syntax for queries.
 	// See: https://cloud.google.com/bigquery/sql-reference/
 	query.QueryConfig.UseStandardSQL = true
```
Comment (broady) [inline bigquery/simpleapp/simpleapp.go]: I think this is default now, can remove?

File: bigquery/simpleapp/simpleapp.go [hunk 2]
```diff
@@ -35,28 +37,41 @@ func main() {
...
 	return query.Read(ctx)
 }
 
-// printResults prints results from a query to the Shakespeare dataset.
+// [START bigquery_simple_app_print]
+type StackOverflowRow struct {
+	Url string `bigquery:"url"`
```
Comment (broady) [inline bigquery/simpleapp/simpleapp.go]: s/Url/URL/

**Comment by broady** [comment]
LGTM, just a couple nits

File: bigquery/simpleapp/simpleapp.go (hunk 1 above)
Comment (tswast) [reply bigquery/simpleapp/simpleapp.go]: You're right. Done.

**Comment by broady** [comment]
Looks like you need to run gofmt. LGTM otherwise.

File: bigquery/simpleapp/simpleapp.go [hunk 3]
```diff
@@ -53,16 +54,16 @@ func query(proj string) (*bigquery.RowIterator, error) {
 				CAST(id as STRING)) as url,
 			view_count
 		FROM ` + "`bigquery-public-data.stackoverflow.posts_questions` " +
-		`WHERE tags like '%google-bigquery%'
+			`WHERE tags like '%google-bigquery%'
```
Comment (broady) [inline bigquery/simpleapp/simpleapp.go]: maybe move the backtick up to the previous line so you can line it all up?

```
FROM ` + "`...` " + `
WHERE ...
```

Comment (tswast) [reply bigquery/simpleapp/simpleapp.go]: Yeah, done.

**Comment by tswast** [comment]
I can't quite tell what's going on with the tests.

I already froze the docs to a commit hash, so this should be safe to merge whenever.

**Pull request merged**